### DISCUSSION TITLE
Excel Spreadsheet With Suggested Names for U(nnnn) Rides

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#23513] [Q&A]
I know there is such a spreadsheet because I have a hardcopy of it.  Problem is the characters are so small I need a magnifying glass to read them.   For each U(nnnn) ride it has a suggested name.  Does anyone know where this is?  I would like to reprint with larger readable characters.   I may have saved the spreadsheet but I cannot find it now on my computer.

**Comment by telk5093** [comment]
https://docs.google.com/spreadsheets/d/1n4wWSwv1LwJIZq5EziJwAsFiHYrSwf69hKJ3vbM_JsQ/edit?gid=0#gid=0
Fortunately I have a link

**Comment by jcurbaniak** [comment]
Thanks so very much.  That is what I needed.  Was finally smart enough to save it this time.
